docker-compose.test.yaml

```diff
@@ -17,7 +17,8 @@ services:
       - ./package.json:/home/node/package.json
     environment:
       AZURE_STORAGE_CONNECTION_STRING: DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://ffc-pay-statement-azurite:10000/devstoreaccount1;
-
+      STATEMENT_RECEIVER_ENDPOINT: http://localhost:3000
```

Comment:
What's the value of setting this to localhost for the test definition?

Localhost will be local to the generator container only which will never resolve.  Would it be better to just mock endpoint in the tests so we don't get any side effect from this?